readme.md

```diff
@@ -198,124 +178,125 @@ Solar System
+* `..`: two dots means "go up to the parent directory"
 
-So if we're in `~/code/work` then the relative path `../personal_projects` means "go up one level to the code folder, then down into my `personal_projects` directory.
+So if we're in `~/code/work`, then the relative path `../personal_projects` means "go up one level to the code folder, then down into my `personal_projects` directory.
 
-**Note**: We can use multiple `..` to go up multiple levels:
+We can use multiple `..` to go up multiple levels:
 
 `../../documents/top_secret/lol_cats/favorites/so_many_kittenz.jpg` would go up two levels, from `~/code/work` to `~` (my home directory), and then down into my favorite lolcat picture.
 
-### Compare images in HTML
+### Compare Images in HTML
 
-Turns out paths are really important in HTML too. If we look at the image tags in the `index.html` file in this repo we'll see this:
+Turns out paths are really important in HTML too. If we look at the image tags in the `index.html` file in this repo we'll see this...
 
-```HTML
+```html
 <img src="../heeler.jpg">
 <img src="firehydrant.jpg">
 <img src="images/troll.png">
 <img src="/Users/andrewkim/wdi/lessons/cli-intro/demo_html/images/troll.png" alt="">
 ```
 
-Of these 4 paths which are relative vs absolute?
+Of these four paths, which are relative vs absolute?
 
-> Note that we have 3 images in this repo. Turns out, it's really important where the file is in your directory when you link them in HTML.
+> We can see here that the `troll.png` photo is linked in two different ways. Which way is better? does it matter?
 
-We can see here that the `troll.png` photo is linked in two different ways. Which way is better? does it matter?
+<!-- AM: Any additional CFUs to add here? -->
 
 --------------------------------------------------------------------------------
 
 ## Common Commands
 
-### Getting help / info (5/110)
+### Getting Help (5 minutes / 1:30)
 
 There are three general ways to get help with a command.
 
-* add `--help` or `-h` to the end of the command. e.g. `brew --help`
-* use the `man` tool (manual), e.g. `man brew`
-* search google
+* Add `--help` or `-h` to the end of the command (e.g., `brew --help`).
+* Use the manual -- or `man` -- tool (e.g., `man brew`).
+* Google!
+
+### Common Command Teachbacks (25 minutes / 1:55)
 
-### Common Command Teachbacks (15/125)
+> 15 minutes preparation. 10 minutes review.
 
-Form groups of ~3 and spend 15 minutes researching and preparing a short demo
-of your command. Focus on:
+Form groups of 3 and spend 15 minutes researching and preparing a short demo of your command. Focus on...
 
-* what it does
-* common uses
-* common flags or arguments
-* any 'gotchas'?
+* What it does.
+* Common uses.
+* Common flags or arguments.
+* Any 'gotchas'?
 
 #### Commands
 
-1. tab completion
-2. ls
-3. cd
-4. touch / mkdir
-5. cp
-6. mv
-7. rm
-8. atom
+1. Tab completion.
+2. `ls`
+3. `cd`
+4. `touch` and `mkdir`
+5. `cp`
+6. `mv`
+7. `rm`
+8. `atom`
```

Comment:
:+1: I included the commands that were included in the "Hungry For More" section towards the end of the lesson plan.